tests/integration/ecdsa_test.py

```diff
@@ -164,11 +157,10 @@ def recover():
             )
         )
 
-    approval_app = blackbox_pyteal(recover, Mode.Application)
-    app_teal = compileTeal(approval_app(), Mode.Application, version=5)
     args = []
-    algod = algod_with_assertion()
-    app_result = DryRunExecutor.dryrun_app(algod, app_teal, args)
+    app_result = PyTealDryRunExecutor(recover, Mode.Application).dryrun(
+        args, compiler_version=5
```

Comment:
Not sure why `compiler_version=5` here

Reply:
I read from the current state of code, we are directly taking in `recover` this `Blackbox` decorated subroutine, rather than compiling from Pyteal to Teal. Then we do need `compiler_version` knowledge in dryrun, I think,.